### DISCUSSION TITLE
Need help making patterns work after installing hacktribe on the blue electribe

### DISCUSSION THREAD

**Discussion by Larifari96** [bangcorrupt/hacktribe#334] [Q&A]
Hello there,

I am an absolutely beginner working with the electribe and with github, so please don't be angry if I've missed an existing discussion regarding my topic.

I have installed hacktribe on my blue electribe. After days of reading through the whole forum, I was also able to create the dump of the blue version's samples as one big .wav (did not got it sliced automatically, but this is not my problem). 

Because I stucked at slice them automatically, I searched for an sample.all file of the red one (I don't care if I have at least the factory samples of the blue or the red one, just wanted some original samples to get started). I found one inside the comments of an youtube video. The file already puts the samples in the slots starting at 501, but in the user samples area. In the file, there are no samples stored in the factory samples section.

I got confused when I wanted to start listening to the factory patterns (I think the patterns are still those of the blue version, starting with 001 Magician and ending with 200 8Bit Template, or do both versions have same factory pattern?). I had to notice that they don't sound like they did before the switch to hacktribe. First, I thought that I adjusted some of the configurations wrong, but am I right that it is because they don't have the correct samples to match anymore?

So, here are my questions:

1. If the patterns are still from the blue version, is there possibility to get the factory patterns of the red one? Because I read in another discussion that when you have both files of one device (sample and pattern files) just the script for switching the slot positions would be necessary to make them match correctly again. 
3. Or does the problem appear because the samples are stored in the user samples section and I have to move them to factory samples section? I use the paid sample manager. If this would solve the problem: Is there a way to move all samples from user to factory samples?
4. If the blue and red one have different factory patterns, why does my blue version still has the patterns its patterns? The "samples" got lost when changing to hacktribe, why the patterns still exist?

I hope you guys can help me. As I said I don't care about getting the blue or the red factory samples and patterns. I just want one version matching those, so I have a basis of samples and factory patterns.

Thank you!

**Comment by bangcorrupt** [comment]
You need the patterns from the sampler, edited to reference the new sample slots.

[FAQ: The factory patterns sound wrong.](https://github.com/bangcorrupt/hacktribe/wiki/FAQ#the-factory-patterns-sound-wrong)

**Reply by Larifari96** [reply]
Is there a possibility to get the patterns of the sampler? Because I only have the synth's pattern. I also tried to use the bonus material patterns from the Korg website (the ones for the sampler) because my idea was that they also needed to use the factory samples.

**Reply by bangcorrupt** [reply]
> bonus material patterns from the Korg

These should work, if you edit them using the script linked in the FAQ.

**Comment by limpbizkit91** [comment]
I'm not sure how to get the patterns to play correctly after installing Hacktribe on the Blue and correctly moving the samples from 519 onwards. Is there a visual guide?

Some samples I've inserted, it gives me a slice effect, for example a 20-second vocal is divided into several pieces organized like this: e.g. 901-01, 901-02, 901-03, 901-04, etc.. (the word slice appears on the screen next to the sample). Do you know why?

**Reply by vanana-old-acc** [reply]
Have u used OE2sSLE or the script ? I know on youtube there's a guide for the sample shifting using OE2sSLE

**Reply by bangcorrupt** [reply]
> moving the samples from 519 onwards.

First sample should be in slot 501.
